### PR TITLE
commits for wt88047

### PR DESCRIPTION
**Pull request by abhishek9650**
/docs/supplementary/devices/wt88047  file needs to be deleted, which was merged by @LNJ2 mistakenly. 
wt88047 file is replaced by updated wt88047.rst which should not give any errors .. atleast from my side!

### CODE REVIEW

File: supplementary/devices/index.rst [hunk 1]
```diff
@@ -26,8 +26,8 @@ All of these should only be document in the exact same way here and **only an ov
    T00F
    titan
    vegeta
+   wt88047
    yuga
-   taoshan
```
Comment (JBBgameich) [inline supplementary/devices/index.rst]: Don't remove other devices please?

Comment (abhishek9650) [reply supplementary/devices/index.rst]: i never removed it

Comment (LNJ2) [reply supplementary/devices/index.rst]: You removed it in this commit, if it was an accident or not: https://github.com/Halium/docs/pull/69/commits/44f740c6e5a86fc2917ce60a9d40ac106ee66518

**Comment by vanyasem** [comment]
@abhishek9650 why haven't you deleted (moved) the wrongly named file in this PR?

File: supplementary/devices/index.rst [hunk 2]
```diff
@@ -453,28 +454,9 @@ Devices
      - Navigation
      - Sensors
      - Other
-   * - :doc:`Sony Xperia L <taoshan>`
```
Comment (JBBgameich) [inline supplementary/devices/index.rst]: You are still deleting taoshan related stuff. Please read the `git diff` next time before committing!

**Comment by LNJ2** [comment]
This device is called "Wingtech Redmi 2" in the [LineageOS wiki](https://wiki.lineageos.org/devices/wt88047/), you called it "Xiaomi Redmi2/Prime", is this the correct name?

**Comment by LNJ2** [comment]
Sorry for pushing to your branch, but I'm trying to speed things a bit up here.

File: supplementary/devices/index.rst [hunk 3]
```diff
@@ -491,6 +492,25 @@ Devices
...
+     - Halium 7.1 and 5.1
+     - 3.10.49
+     - Snapdragon 410 (MSM8916)
+     - Quad-Core ARMv8 (4x1.3Ghz)
+     - Adreno 306
+     - 1GB/2GB LPDDR3
+     - 8GB/16GB + MicroSd upto 64GB
+     - WiFi, Bluetooth4.1, DLNA
+     - 2 Mp
+     - 8 Mp
```
Comment (LNJ2) [inline supplementary/devices/index.rst]: `MP`, not `Mp`

Comment (abhishek9650) [reply supplementary/devices/index.rst]: thank you, :)

File: supplementary/devices/wt88047.rst [hunk 4]
```diff
@@ -24,14 +24,12 @@ Distributions
      - What works
      - What doesn't work
    * - `WT88047 on LineageOS Wiki <https://wiki.lineageos.org/devices/wt88047>`_
-     - `LineageOS\android_device_wingtech_wt88047 <https://github.com/LineageOS/android_device_wingtech_wt88047>`_
-     - `LineageOS\android_device_wingtech_msm8916-common <https://github.com/LineageOS/android_device_wingtech_msm8916-common>`_
+     - `LineageOS\android_device_wingtech_wt88047 <https://github.com/LineageOS/android_device_wingtech_wt8804>`_
      - `LineageOS\android_kernel_wingtech_msm8916 <https://github.com/LineageOS/android_kernel_wingtech_msm8916>`_ based on v3.10.49
      - see device page
      - see device page
    * - `WT88047 on Halium projectmanagement <https://github.com/Halium/projectmanagement/issues/36>`_
```
Comment (LNJ2) [inline supplementary/devices/wt88047.rst]: Please remove this: there should be only LineageOS and other ports, not halium itself.

Comment (LNJ2) [reply supplementary/devices/wt88047.rst]: The problem is,  it's also wrong on the pme's page, not your fault.

Comment (vanyasem) [reply supplementary/devices/wt88047.rst]: @LNJ2 noted, will fix that as soon as i can

File: supplementary/devices/index.rst [hunk 5]
```diff
@@ -491,6 +492,25 @@ Devices
      - GPS + GLONASS + (Beidou) Navigation **??**
      - Ambient light sensor, Proximity sensor, Motion G-sensor, Compass sensor, Gyro sensor, Magnetic sensor, Fingerprint sensor, Sensor Hub - **??**
      - Other **??**
+   * - :doc:`Xiaomi Redmi2/Prime <wt88047>`
+     - wt88047
+     - Halium 7.1 and 5.1
```
Comment (LNJ2) [inline supplementary/devices/index.rst]: I can't find anything about the 5.1 port, is this correct?

**Pull request merged**